tests/onnx/test_onnx_model_export.py

```diff
@@ -424,3 +443,17 @@ def test_model_log_without_specified_conda_env_uses_default_env_with_expected_de
         conda_env = yaml.safe_load(f)
 
     assert conda_env == mlflow.onnx.get_default_conda_env()
+
+
+@pytest.mark.large
+def test_OnnxModelWrapper_predit_handles_list_output(onnx_sklearn_model,  model_path, data):
```

Comment:
Nit: typo - `predit` -> `predict`.

rather than referring to `OnnxModelWrapper` in the test case name, I think we should refer to the pyfunc representation of the ONNX model. I.e.:
```
test_pyfunc_predict_supports_models_with_list_outputs
```

Reply:
Done